src/views/User.vue

```diff
@@ -172,7 +171,35 @@ const anyCourseOptedInOrScheduled = computed(() => {
 })
 
 onMounted(() => {
```

Comment:
The diff below is not as complicated as it seems. I simply moved the `getUser` call off into its own function: `refreshUser()`.